pkg/webhooks/server.go

```diff
@@ -29,6 +29,14 @@ type DebugModeOptions struct {
 	DumpPayload bool
 }
 
+// ExceptionOptions holds flags to enable PolicyExceptions and in which namespace
+type ExceptionOptions struct {
+	// EnablePolicyException enables/disables PolicyExceptions
+	EnablePolicyException bool
+	// Namespace is the defined namespace
+	Namespace string
+}
+
```

Comment:
Now we can move this struct closer to the handler

Reply:
@eddycharly it leads to an import cycle

Reply:
Impossible

Reply:
Don't you mean moving it close to the handler here?

https://github.com/kyverno/kyverno/blob/fd0699c6d4411d701c3108147b2f6bf4ddffe9f4/pkg/webhooks/exception/validate.go#L14

Reply:
@eddycharly 
/validation/exception -> /webhooks/exception
/webhooks/exception -> /validation/exception

Reply:
> Don't you mean moving it close to the handler here?

Yes

Reply:
An import cycle occurs

Reply:
Impossible, this should be referenced only from main...